jest.config.ts

```diff
@@ -15,7 +15,8 @@ const config: Config.InitialOptions = {
     'graphql/index.tsx',
     '__tests__/utils/',
     'prisma/',
-    'scripts/'
+    'scripts/',
+    'discordAuth.ts'
```

Comment:
could we write tests for this instead of excluding it from tests? I see a few places where it could go wrong.

For example, when you call `getUserInfoFromRefreshToken`, you want to make sure:

1. `if (!updatedRefreshToken) throw new Error('refresh token invalid')`. 
2. `updateUserRefreshToken` is called

I think if you tested these 2 you should have 100% coverage on this file because that's the only branching.

Reply:
removed